_posts/2018-10-29-go.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Go over Nabla- App Safety meets Host Isolation
```

Comment:
Hmm, i took quite a while to parse this, i thought it was Nabla-App. Perhaps use a colon here?